docs/hints-for-use-with-docker.md

```diff
@@ -57,3 +57,24 @@ To authenticate with a private Git repository, ORT uses the (semi)standardized `
    ```
 
    **Important:** Ensure that the `.netrc` file has been created at `netrc_folder_path` before running the command, otherwise Docker will create a folder `.netrc` inside the container, instead of mounting a single file.
+
+It is also possible to use `git config` to setup access to private repositories.
+
+1. Configure your gitconfig with your github credentials
+
+    ```shell
+    git config --global url."https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com".insteadOf /"https://github.com"
+    ```
+
+    ```shell
+    cat ~/.gitconfig
+    [url "https://<username>:<password>@github.com"]
+	         insteadOf = /https://github.com    
+    ```
```

Comment:
I think we should not document pure Git behaviour in this documentation.
A user does not need to run the `git config` command to create the .gitconfig file, they could also create it manually.

````suggestion
1. Configure the `.gitconfig` with the repository credentials. E.g. Github:
    ```
    [url "https://<username>:<password>@github.com"]
             insteadOf = https://github.com    
    ```
````

Reply:
Thanx, implemented changes as suggested.